pkgs/by-name/rl/rlottie/package.nix

```
@@ -3,40 +3,28 @@
   stdenv,
   fetchFromGitHub,
   fetchpatch,
-  cmake,
+  meson,
   ninja,
   pkg-config,
 }:
 
-stdenv.mkDerivation rec {
+stdenv.mkDerivation {
   pname = "rlottie";
-  version = "0.2";
+  version = "0.2-unstable-20251001";
```

Sorry, abit tired… The timestamp also needs dashes.

```suggestion
  version = "0.2-unstable-2025-10-01";
```

(also, please squash 7444f2025709276780126051f560dd70495d4b54 and whatever commit id this change ends up being into 27a56d379455813baee069f3b7684e36c445af01)